src/exampleTypes/Facet.js

```diff
@@ -11,7 +11,7 @@ let toggleElement = (check, val, arr = []) =>
   (check ? _.pull : F.push)(val, arr)
 
 export default injectTreeNode(
-  observer(({ tree, node, hide = {}, TextInput = 'input' }) => (
+  observer(({ tree, node, values, hide = {}, TextInput = 'input' }) => (
```

Comment:
This shouldn't be necessary. If you want to set values from the Facet component you should be able to already as this inject tree node reads it off the reactor config from the facet example type

Reply:
In fact, this code won't actually even check the values on the facet because it doesn't call mutate with values. Was the goal to enable setting `values` with `path` usage instead of group where the component creates the node? If that doesn't work then the right approach is to extend injectTreeNode to handle that case as well if we think it's valuable (though I think you'd either have the components completely own setting up the nodes or you'd set them up in code, not have a weird mix where you set some node values from component props which is pretty confusing)